packages/@okta/vuepress-site/docs/guides/auth-js-redirect/main/index.md

```diff
@@ -2,13 +2,13 @@
 title: Sign users in to your SPA using redirect and Auth JS
 ---
 
-The [Okta JavaScript Auth SDK](https://github.com/okta/okta-auth-js) (Auth JS) helps implement many Web Authentication solutions for both the [redirect and embedded models](/docs/concepts/redirect-vs-embedded/). This guide creates a simple redirect authentication solution using Auth JS. Once created, you can drop the solution into just about any front-end or server-side web application.
+The [Okta JavaScript Auth SDK](https://github.com/okta/okta-auth-js) (Auth JS) helps implement many Web Authentication solutions for both the [redirect and embedded models](/docs/concepts/redirect-vs-embedded/). This guide creates a simple redirect authentication solution using Auth JS. Once created, you can drop the solution into just about any front-end or server-side web app.
 
 ---
 
 #### Learning outcomes
 
-* Understand how to implement a simple redirect sign-in using Auth JS.
+* Understand how to implement a simple redirect sign in using Auth JS.
```

Comment:
we try to avoid using sign in / sign-in as a noun. Is there another way we could say this?